test/parallel/taskCompare/elliot/taskSpawn.graph

```diff
@@ -1,5 +1,5 @@
-perfkeys: Elapsed time:, Elapsed time:, Elapsed time:
-graphkeys: forall, coforall, for+begin
-files: empty-forall.dat, empty-coforall.dat, empty-for+begin.dat
+perfkeys: Elapsed time:, Elapsed time:, Elapsed time:, Elapsed time:
+graphkeys: forall, coforall, for+begin, omp parallel-for
+files: empty-forall.dat, empty-coforall.dat, empty-for+begin.dat, empty-omp-parallel-for.dat
 graphtitle: Empty Task Spawn Timings (500,000 x maxTaskPar)
```

Comment:
Ah, I missed that there was already a PERFEXECOPTS.  I think this is all set then.